src/main/java/ithillel/lesson7/FileNavigator.java

```diff
@@ -0,0 +1,72 @@
+package ithillel.lesson7;
+
+import java.util.*;
+
+public class FileNavigator{
+    private final Map<String, List<FileData>> navigator = new HashMap<>();
+    private List<FileData> sorteredList;
+
+    public void add(String pathDir, String fileName, int size, String pathFile){
```

Comment:
```suggestion
    public void add(String pathDir, FileData file){
```
поговоримо на уроці чому так